fiatconnect-api.md

```diff
@@ -2352,6 +2353,34 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 9.3.2.6. `PIXAccount`
+
+`PIXAccount`  egg. [PIX](https://www.bcb.gov.br/estabilidadefinanceira/pix) is the Brazilian instant payment. The payment method created by the Central Bank (BC) in which funds are transferred between accounts in a few seconds, at any time or day. It's practical, fast and safe. Pix can be carried out from a checking account, savings account or prepaid payment account.
```

Comment:
```suggestion
`PIXAccount`  egg. [PIX](https://www.bcb.gov.br/estabilidadefinanceira/pix) is a Brazilian instant payment method created by the Central Bank (BC) in which funds are transferred between accounts in a few seconds, at any time or day. Pix transfers can be carried out from a checking account, savings account or prepaid payment account.
```
Trimmed this down a little-- let's keep the description focused on what's needed for operating or consuming a FiatConnect API.